common/djangoapps/student/views/dashboard.py

```diff
@@ -472,6 +473,21 @@ def get_dashboard_course_limit():
     return course_limit
 
 
+@login_required
+@ensure_csrf_cookie
+def send_account_activation_email(request):
+    try:
+        user = request.user
+        profile = UserProfile.objects.get(user=user)
+        if not user.is_active:
+            from common.djangoapps.student.views import compose_and_send_activation_email
+            compose_and_send_activation_email(user, profile)
+
+        return HttpResponse('success', status=200)
```

Comment:
I guess just status is fine no need for a message. The same goes for the 500 response.